schema/transportation/segment.yaml

```diff
@@ -132,7 +132,9 @@ properties:
             - parkingAisle # service road intended for parking
             - driveway # service road intended for deliveries
             - pedestrian
-            - footway
+            - path
+            - sidewalk
+            - crosswalk
```

Comment:
Are we ready to take this out?

Asking because the combobulator [still maps highway=footway through to the segment](https://github.com/OvertureMaps/schema-wg/blob/transportation_combobulation/transportation/combobulation/notebook/_08_road_property.py#L246-L265), which means that it will combobulate data that doesn't follow this schema.